File: source java/organism/plants/Grass.java
package organism.plants;

import java.awt.Color;
import organism.manager.Organisms;
import utilities.position.Position;
import utilities.savingsystem.Loader;
import virtualworld.World;

public class Grass extends Plant
{
    private static final int STRENGTH = 0;

    public Grass(Position position, World world) 
    {
        super(STRENGTH, position, world);
    }
    
    public Grass(Loader loader, World world, Position position)
    {
        super(loader, world, position);
    }
    
    @Override
    public Organisms GetEnum() 
    {
        return Organisms.GRASS;
    }
    
    @Override
    public Color GetColor()
    {
        return Color.green;
    }
    
    @Override
    public String toString()
    {
        return "Grass";
    }
}

File: source python/Organisms/Plants/Grass.py
from Application.Organisms.Plants.Plant import Plant
from Application.Utilities import Colors


class Grass(Plant):

    STRENGTH = 0

    def __init__(self, position, world, **kwargs):
        super().__init__(Grass.STRENGTH, position, world, **kwargs)

    @staticmethod
    def __str__():
        return "Grass"

    def GetColor(self):
        return Colors.GREEN

    def GetLetter(self):
        return "T"